inner-scripts/kubestellar

```diff
@@ -357,21 +403,9 @@ if [ $subcommand == stop ]; then
    exit 0
 fi
 
-# Check if kcp is running
-if [ "$local_kcp" == true ] && [ $(process_running kcp) != "running" ]; then
-    echo "kcp is not running - please start it ...."
-    exit 1
-fi
-
-let tries=1
-while ! ( kubectl get workspaces && kubectl ws root:compute ); do
-    if (( ( tries & (tries-1) ) == 0 ))
-    then echo "$0: At $(date), kcp server not running yet or root:compute workspace not yet ready, waiting $(( tries * 15 )) sec..." >& 2
-    fi
-    let tries=tries+1
-    sleep 15
-done
-
+# TODO: remove kcp dependency
+KUBECONFIG=/home/kubestellar/.kcp/admin.kubeconfig
```

Comment:
This looks odd to me. I do not have `/home/kubestellar` on my Mac.

Reply:
That's a really good point.  We need to access the kcp config here because we have a dependency that will be removed once the kube-bind PR is merged.  For now, I'll add code to set the kcp config using the secret in the space provider.